src/apps/versionRequirements.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import semver from "semver";
+import type { DeviceModelId } from "@ledgerhq/devices";
+import { getDependencies } from "./polyfill";
+import { getEnv } from "../env";
+
+export function shouldRecommendUpgrade(
+  deviceModel: DeviceModelId,
+  appName: string,
+  appVersion: string
+) {
+  if (getEnv("DISABLE_APP_VERSION_REQUIREMENTS")) return false;
+  const deps = getDependencies(appName);
+  if (deps.includes("Bitcoin") && deviceModel !== "blue") {
+    // https://donjon.ledger.com/lsb/010/
+    return !semver.satisfies(appVersion, ">= 1.4.0");
+  }
```

Comment:
Would it be possible to add a shouldUpgrade for cosmos to be `>=2.13` ? (regarding the FW app bug we saw recently)

Reply:
We can yeah. I think let's wait we see how we use this as contract might change in future. Today it might be highly related to recommending upgrade because security requirement.